Source/Core/Common/Assert.h

```diff
@@ -13,7 +13,7 @@
 #define _assert_msg_(_t_, _a_, _fmt_, ...)                                                         \
   if (!(_a_))                                                                                      \
   {                                                                                                \
-    if (!PanicYesNo(_fmt_, __VA_ARGS__))                                                           \
+    if (!PanicYesNo(_fmt_ "\nDo you want to continue running Dolphin?", __VA_ARGS__))                                                           \
```

Comment:
When I suggested this wording on IRC, I missed the fact that we already had a similar sentence for `_assert_` ("Ignore and continue?"). It would be nice to have the two match. I would also prefer to have two newlines instead of one, so that there's a blank line between the two parts of the message.